response_operations_ui/error_handlers.py

```diff
@@ -14,10 +16,22 @@
 def api_error(error):
     logger.error('Api failed to retrieve required data', url=error.url,
                  status_code=str(error.status_code), exc_info=error)
-    return "FAIL"
+    return redirect(url_for('error_bp.server_error_page',
+                            _external=True,
+                            _scheme=getenv('SCHEME', 'http')))
 
 
 @app.errorhandler(RequestException)
 def connection_error(error):
     logger.error('Failed to connect to external service', url=error.request.url)
-    return "FAIL"
+    return redirect(url_for('error_bp.server_error_page',
+                            _external=True,
+                            _scheme=getenv('SCHEME', 'http')))
+
+
+@app.errorhandler(Exception)
+def server_error(error):  # pylint: disable=unused-argument
+    logger.exception('Uncaught exception generated', exception=error)
```

Comment:
don't log the exception twice. Using logger.exception already logs out the exc_info if any is present

Reply:
Agreed.  Removed it.